front/src/applications/stdcm/components/StdcmHelpModule/index.ts

```diff
@@ -0,0 +1,3 @@
+import StdcmHelpModule from './StdcmHelpModule';
+
+export default StdcmHelpModule;
```

Comment:
You can simply reeport default. 

```suggestion
export { default } from './StdcmHelpModule';
```

same for all your index.ts

Reply:
Hum I'm not sure it will be a default export like this

Reply:
named reexports are using `as`  (`export { default as StdcmHelpModule } from ''`)

Reply:
I get a warning if I do it that way : 
Prefer default export on a file with single export. eslint(import/prefer-default-export)

Reply:
with `export { default } from './StdcmHelpModule';` ?

Reply:
![Screenshot_20250219_112543](https://github.com/user-attachments/assets/3b2cd5c8-f505-440c-bb7a-d282bf386b65)

Reply:
@clarani @emersion @SharglutDev 
I think we can turn that off, any reason to disallow that ?

![Screenshot_20250219_113523](https://github.com/user-attachments/assets/acad8827-da46-4c60-8ac0-bd7e85b5beb5)

Reply:
No strong opinion about that :)

Reply:
Hum I find it more difficult to spot the default export at the first glance. What do you think ? I think it does not cost much to import and then re-export it as default, and it's more readable to me

Reply:
imo the shorter version is better, it’s just a habit to take.

Reply:
We could also `export * from …` to re-export everything if we wanted to.

Reply:
`export * from …` only export named stuff though. 

we need both lines if we have default + named 

```
export * from "./abcd";
export { default } from "./abcd";
```

Reply:
I think we can get this PR across this way, and we could create another ticket for this topic.